http-troubleshooting.md

```diff
@@ -0,0 +1,32 @@
+# Kubecost HTTP Response Code Troubleshooting
+
+If you have experienced an HTTP response code while using Kubecost, consult this doc for a possible fix.
+
+You will likely encounter these response codes in the browser, when performing REST requests through a client like cURL, or when viewing the logs `kubectl logs deploy/kubecost-cost-analyzer -c cost-analyzer-frontend -n kubecost`.
```

Comment:
In Thomas' doc, this is all written as one singular log, is it supposed to several? If it's just the one, I need to correct the plural "logs".